pkg/internal/venafi/sign.go

```diff
@@ -29,7 +29,7 @@ import (
 	"github.com/jetstack/cert-manager/pkg/util/pki"
 )
 
-// ErrCustomFieldsType provides a common error structure for a fivalid custom field types
+// ErrCustomFieldsType provides a common error structure for an invalid Venafi custom field type
```

Comment:
I'm really uncertain about this change.

Reply:
your edit is correct.. I'm not sure *how* `fivalid` got through review 😆 thanks!